sale_order_carrier_auto_assign/__manifest__.py

```diff
@@ -4,10 +4,10 @@
     "name": "Sale Order Carrier Auto Assign",
     "summary": "Auto assign delivery carrier on sale order confirmation",
     "version": "16.0.1.1.0",
-    "development_status": "Alpha",
```

Comment:
If it's not `Alpha`, is it `Beta`/`Mature`/`Production/Stable`?
I think we should not remove it, I do not know the default value...?